integration-test/fixture.ts

```diff
@@ -8,6 +37,7 @@ export const test = base.extend<{
     page.on("pageerror", (error) => {
       expect(error.stack || error).toBe("no error");
     });
+    page.route("**", (route) => route.continue());
```

Comment:
this prevents the playwright http cache to kick in in test development

Reply:
Maybe add a comment to that effect (unless this is a well established Playwright convention)?

Reply:
good point :) And I have no idea ^^